OpenRA.Game/Traits/Selectable.cs

```diff
@@ -9,58 +9,18 @@
 #endregion
 
 using System.Collections.Generic;
-using System.Drawing;
 using System.Linq;
-using OpenRA.Graphics;
 
 namespace OpenRA.Traits
 {
 	public class SelectableInfo : ITraitInfo
```

Comment:
You can make shorten this to `TraitInfo<Selectable>` if this is otherwise empty.